src/utils/auto-box-collider.js

```diff
@@ -7,8 +24,9 @@ export function getBox(entity, boxRoot) {
 
   entity.object3D.updateMatrices(true, true);
   boxRoot.updateMatrices(true, true);
+  boxRoot.updateMatrixWorld(true);
```

Comment:
This is surprising. I thought that `boxRoot.updateMatrices(true, true)` will force a world update. Why is `updateMatrixWorld(true)` necessary here?

(The forced updates above are already surprising by the way. I don't know why we need to force local or world updates here.)